_about/release-cadence.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Build & Release Cadence
+description: How we manage, number, and support Istio releases.
+weight: 5
+---
+
+We produce new builds of Istio on a daily basis.
+Once a week, one of these daily build undergoes additional
+qualification and gets anointed the weekly release. Once a month, one of the weekly releases undergo
+additional qualifications and gets anointed as the monthly release. Once a quarter or so, one of the monthly
+releases undergoes additional qualifications and gets anointed as a long term supported (LTS) release.
```

Comment:
I'm not familiar with the additional qualification for weekly release.  Is this doc-ed anywhere?

Reply:
That came from https://github.com/istio/istio/blob/master/release/README.md and from https://docs.google.com/document/d/1_SNEDOSGyw69UnLC9kYY7bTfxVszTE-W-mLdtPVJuOA/edit

Reply:
Weekly release has not been implemented.
And I will send a PR to update the ETA in https://github.com/istio/istio/blob/master/release/README.md

Reply:
Weekly release pipeline has not been implemented.

I will send a PR to update the ETA in https://github.com/istio/istio/blob/master/release/README.md.